### PR TITLE
[dhcp_relay] Update CLI reference document and add a new API for ip address type

### PR DESCRIPTION
**Pull request by shlomibitton**
Signed-off-by: Shlomi Bitton <shlomibi@nvidia.com>

#### What I did
Update CLI reference document following PR : https://github.com/Azure/sonic-buildimage/pull/8211
Add a new API on utilities_common to get IP type.

#### How I did it
- Update doc/Command-Reference.md with new DHCP CLI.
- Add ipaddress_type API to utilities_common/cli.py

#### How to verify it
- Build an image with PR https://github.com/Azure/sonic-buildimage/pull/8211 and this PR.
- Run DHCP CLI commands

### CODE REVIEW

**Comment by shlomibitton** [comment]
@tahmed-dev can you please review this PR?
Thanks

**Comment by anshuv-mfst** [comment]
Hi @tahmed-dev - could you please review and approve asap, this is needed in 202106 release

File: utilities_common/cli.py [hunk 1]
```diff
@@ -200,6 +201,17 @@ def is_ipaddress(val):
         return False
     return True
 
+def ipaddress_type(val):
```
Comment (tahmed-dev) [inline utilities_common/cli.py:204]: is `ipaddress_type` used anywhere?

Comment (shlomibitton) [reply utilities_common/cli.py:204]: Yes, on the other PR for the CLI:
https://github.com/Azure/sonic-buildimage/pull/8211
This is why this PR is a dependency for the other.

**Comment by liat-grozovik** [comment]
@tahmed-dev kindly reminder to review following the above reply

**Comment by liat-grozovik** [comment]
@tahmed-dev kindly remider

**Review by tahmed-dev — APPROVED** [review]

**Pull request merged**